README.md

```diff
@@ -1 +1,31 @@
 # UITeamB
+
+Creating a local version of the project
+---
+1. Run "Git clone {repository link}"
+2. Run "Git pull" to bring files up to date 
+
+How to start the application
+---
+
+1. Open the terminal
+2. Run `npm install` to build the application
+3. Start application with `npm start`
```

Comment:
UI works only when API is running in the background. I think we should add one new point with reference to an API github repo.